quickstarts/aws/aws-rekognition/config.yml

```
@@ -0,0 +1,30 @@
+slug: aws-rekognition
+description: |-
+  ## Monitor your AWS Rekognition and AWS Rekognition Operation Metrics
+
+  With New Relic's Amazon Rekognition integration, you now can see all your Rekognition data on New Relic Instant Observability Platform.
+
+  ### Get started!
+
+   Check out the [documentation](https://docs.newrelic.com/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-rekognition-monitoring-integration/) to learn more about New
+   Relic monitoring and integration with Amazon Rekognition.
+
+summary: Use New Relic - Amazon Rekognition Integration to analyse your rekognition data with New Relic
```

```suggestion
summary: Use New Relic - Amazon Rekognition Integration to analyze your rekognition data with New Relic
```

We standardize on American spelling

---

addressed